widget/hyperlink_test.go

```diff
@@ -63,3 +64,21 @@ func TestHyperlink_SetUrl(t *testing.T) {
 	hyperlink.SetURL(sURL)
 	assert.Equal(t, sURL, hyperlink.URL)
 }
+
+func TestHyperlink_CreateRendererDoesNotAffectSize(t *testing.T) {
+	url, err := url.Parse("https://github.com/fyne-io/fyne")
+	require.NoError(t, err)
+	link := NewHyperlink("Test", url)
+	link.Resize(link.MinSize())
+	assert.NotEqual(t, fyne.NewSize(0, 0), link.Size())
+	assert.Equal(t, link.Size(), link.MinSize())
+	size := link.Size()
```

Comment:
Consider moving this up two lines, and then you can use the variable in the asserts